### DISCUSSION TITLE
Merging of two triples database

### DISCUSSION THREAD

**Discussion by qingshuizheng** [ahyatt/ekg#145] [Ideas]
Hello @ahyatt ,
I recently just have a career shifting, now I have 2 laptops, one at home, one at work.
Is there an easy way to combine different triples.db from two laptops? 

Thanks!
Zheng

**Comment by ahyatt** [comment]
Interesting question!  I think in theory you should be able to do this.  There are a few instances where the database would have conflicts, but I think it's workable to just have one overwrite the other.  Make sure both are upgraded to the same version of ekg.

You can try a solution similar to what is suggested here: https://stackoverflow.com/questions/9349659/fastest-way-merge-two-sqlite-databases.  If this works, please report back, I'd like to know!  And don't forget to back up your db before trying anything as risky as this!

**Reply by qingshuizheng** [reply]
Thanks I'll report back later.
